### DISCUSSION TITLE
如何使用 quic_config_set_tls_selector 设置 tls配置选择器

### DISCUSSION THREAD

**Discussion by nixwl** [Tencent/tquic#141] [Q&A (问题和回答)] [answered]
下面是 quic_config_set_tls_selector 的 C/C++ API（来自官方API文档）
![image](https://github.com/Tencent/tquic/assets/80773211/a5495968-a439-423b-8b08-d34e22c714ca)

我想为 quic 服务器配置  tls配置选择器 ，但我不太清楚 这个参数 `quic_tls_config_select_context_t ` 该如何设置

**Comment by xiaofei0800** [comment, marked as answer]
可参考[服务端C示例](https://github.com/tquic-group/tquic-example-c/blob/main/simple_server.c)中的quic_config_set_tls_selector使用方法，示例中select方法仅返回了默认的SSL_CTX，可对select方法进行改进，例如根据输入的server_name返回不同的SSL_CTX，用于服务端根据SNI选择证书场景。
